project/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,127 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class User {
+  private final long userId = 10000000L;
+  private String name;
+  private String email;
+  private Set<String> interests;
+  private Set<String> skills;
+  private Set<Event> eventsHosting;
+  private Set<Event> eventsParticipating;
+  private Set<Event> eventsVolunteering;
+
+  public static class Builder {
+    // Required parameters
+    private String name;
+    private String email;
+
+    // Optional parameters
+    private Set<String> interests = new HashSet<>();
+    private Set<String> skills = new HashSet<>();
+    private Set<Event> eventsHosting = new HashSet<>();
+    private Set<Event> eventsParticipating = new HashSet<>();
+    private Set<Event> eventsVolunteering = new HashSet<>();
+
+    public Builder(String name, String email) {
+      this.name = name;
+      this.email = email;
+    }
+
+    public Builder interests(Set<String> interests) {
```

Comment:
setInterests. And for the following functions.

Reply:
Done